modules/API/API.cpp

```diff
@@ -474,10 +475,11 @@ void API::init() {
         });
 
         std::string cmd_set_limit = cmd_base + "set_limit_amps";
-        this->mqtt.subscribe(cmd_set_limit, [&evse](const std::string& data) {
+        this->mqtt.subscribe(cmd_set_limit, [&evse, &r_evse_manager_energy_sink = this->r_evse_manager_energy_sink,
```

Comment:
Do I get it right, that all EVSE managers here will get the same limit?  Is this useful?

Reply:
No, the `cmd_set_limit` topic is different for every EVSE.

Reply:
Right, missed that `cmd_base` gets set for each evse